lib/Makefile.am

```diff
@@ -16,7 +16,7 @@
 #  See the License for the specific language governing permissions and
 #  limitations under the License.
 
-SUBDIRS = ts records . tsconfig cppapi
+SUBDIRS = . ts records tsconfig cppapi
```

Comment:
Why not treat yamlcpp as a subdirectory?

Reply:
It is better to have it as a target, so it doesn't pick up install and check targets.  Right now those don't do anything, but they might in the future.

Reply:
Correct, but it can be optional by adding it to SUBDIRS like BUILD_PERL_LIB below.

Reply:
Yes I agree @bryancall.

Reply:
Ultimately, though it was not put in as a `SUBDIR` in order to avoid clang analysis?

Reply:
Yes. @SolidWallOfCode